### PR TITLE
docs(action): MongoDB documentation

### PR DESCRIPTION
**Pull request by bessonm**
https://bessonm.github.io/actions/mongo/

### CODE REVIEW

File: docs/actions/mongo.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+!!! important "Target Configuration"
+    For all actions, the target should have a property `databaseName`
```
Comment (nbrouand) [inline docs/actions/mongo.md:2]: please provide a target example as others

Comment (bessonm) [reply docs/actions/mongo.md:2]: done

File: docs/actions/mongo.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    | `count` | long   |
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+MongoCountTask(
+    target = "mongo_target",
+    collection = "myCollection",
+    query = "my query"
```
Comment (nbrouand) [inline docs/actions/mongo.md:42]: for all queries, you could set a real mongodb query, for example :
collection = "ghibli_movies",
query = "{ name: { $in: [ 'Pom Poko', 'Spirited Away' ] } }"

File: docs/actions/mongo.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+``` kotlin
+MongoInsertTask(
+    target = "mongo_target",
+    collection = "myCollection",
+    document = "my document"
+)
+```
+
+# List
+
```
Comment (nbrouand) [inline docs/actions/mongo.md:127]: Specify what it lists

File: docs/actions/mongo.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    No output
+
+### Example
+
+=== "Kotlin"
+``` kotlin
+MongoInsertTask(
+    target = "mongo_target",
+    collection = "myCollection",
+    document = "my document"
```
Comment (nbrouand) [inline docs/actions/mongo.md:122]: same as query for all document/update fields: 
```
document = """
{
    id: "123456",
    title: "My Neighbor Totoro",
    original_title: "となりのトトロ",
    original_title_romanised: "Tonari no Totoro",
    description: "Two sisters move to the country with their father in order to be closer to their hospitalized mother, and discover the surrounding trees are inhabited by Totoros, magical spirits of the forest. When the youngest runs away from home, the older sister seeks help from the spirits to find her.",
    director: "Hayao Miyazaki",
    producer: "Hayao Miyazaki",
    release_date: 1988,
    running_time: 86,
}
"""

File: docs/actions/mongo.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+=== "Inputs"
+
+    | Required | Name           | Type          | Default |
+    |:--------:|:---------------|:--------------|:-------:|
+    |    *     | `target`       | String        |         |
+    |    *     | `collection`   | String        |         |
+    |    *     | `filter`       | String        |         |
+    |    *     | `update`       | String        |         |
+    |          | `arrayFilters` | List<String\> |         |
```
Comment (nbrouand) [inline docs/actions/mongo.md:159]: specify only since @3.5.12 mongodb version

**Review by nbrouand — CHANGES_REQUESTED** [review]

**Review by nbrouand — APPROVED** [review]

**Review by KarimGl — APPROVED** [review]

**Pull request merged**